### PR TITLE
chainlink models

### PR DESCRIPTION
**Pull request by austinFlipside**
adds a schema for chainlink oracle feeds

### CODE REVIEW

File: models/chainlink/silver/silver__chainlink_feeds.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+version: 2
+models:
+  - name: silver__chainlink_feeds
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - ID
+
+    columns:
```
Comment (drethereum) [inline models/chainlink/silver/silver__chainlink_feeds.yml:9]: Can we add CONTRACT_ADDRESS tests for NOT NULL and potentially format (could use the token_address_match test)?

Comment (austinFlipside) [reply models/chainlink/silver/silver__chainlink_feeds.yml:9]: added

**Review by drethereum — APPROVED** [review]
